bundle/manifests/windows-exporter_v1_service.yaml

```diff
@@ -5,11 +5,15 @@ metadata:
   labels:
     name: windows-exporter
   name: windows-exporter
+  annotations:
+    service.alpha.openshift.io/serving-cert-secret-name: windows-machine-config-operator-tls
```

Comment:
This annotation seems unrelated to the servicemonitor addition. If that is true, moving them to separate commits would be best.

Reply:
moved to the commit where this secret is used.